### PR TITLE
[COM-76] Added ZRC2 support to the multisig web application.

### PR DESCRIPTION
**Pull request by ashishnair-zil**
1.) Created a new multisig smartcontract which supports both ZRC2 and nativeZil.
2.) Add ZRC2 funds the multisig wallet.
3.) Import ZRC2 tokens in multisig wallet.
4.) Transfer: Transfer ZRC2 token from Multisig to different user.
5.) TransferFrom: Transfer ZRC2 token from One wallet to other wallet if the wallet address has given allowance to the multisig wallet.
6.) IncreaseAllowance: Give access to a specific wallet to control funds of multisig
7.) DecreaseAllowance: Reducing allowance of a specific wallet to control funds of multisig.
8.) Burn: Reduce total supply of a ZRC2 token.
9.) Mint: Increase total supply of a ZRC2 token
10.) Bugs fixes

Jira Ticket: https://zilliqa-jira.atlassian.net/browse/COM-76

### CODE REVIEW

File: src/components/AddToken.vue [hunk 1]
```diff
@@ -47,6 +47,7 @@ export default {
     return {
       isLoading: false,
       error: false,
+      errorMsg: "Incorect token",
```
Comment (inamirandaz) [inline src/components/AddToken.vue:50]: typo, missing 'c' in Incorrect.

**Comment by ashishnair-zil** [comment]
Closing this PR as [PR 29](https://github.com/Zilliqa/zilliqa-multisig-wallet/pull/29) include this changes

**Comment by ashishnair-zil** [comment]
Closing this PR as [PR 29](https://github.com/Zilliqa/zilliqa-multisig-wallet/pull/29) include this changes

File: src/components/AddToken.vue (hunk 1 above)
Comment (ashishnair-zil) [reply src/components/AddToken.vue]: Done. Fixed the typo error

**Comment by inamirandaz** [comment]
@ashishnair-zil can you also add `us-97`  in the description of the PR if the fix for this bug is included?

File: src/views/Wallet.vue [hunk 2]
```diff
@@ -144,29 +158,35 @@ export default {
...
-    // },
     onCancelAddFunds() {
       this.addFunds = false;
     },
-    // onCancelAddToekn() {
-    //   this.newTransaction = false;
-    //   this.addFunds = false;
-    //   this.addToken = false;
-    // },
+    onCancelAddToekn() {
```
Comment (inamirandaz) [inline src/views/Wallet.vue:177]: @ashishnair-zil typo on the token

**Comment by inamirandaz** [comment]
Note (no action item needed): the support for ZRC2 includes all the required transitions and 2 optional transitions (Mint and Burn), other optional transitions are not support as well as custom transitions that contract owners add in their ZRC2 contracts.

File: src/views/Wallet.vue (hunk 2 above)
Comment (ashishnair-zil) [reply src/views/Wallet.vue]: Done

**Review by inamirandaz — APPROVED** [review]
lgtm

**Pull request merged**